recipe-server/normandy/recipes/models.py

```diff
@@ -560,7 +560,7 @@ def get_absolute_url(self):
         return reverse('action-detail', args=[self.name])
 
     def compute_implementation_hash(self):
-        return hashlib.sha1(self.implementation.encode()).hexdigest()
+        return sri_hash(self.implementation.encode(), url_safe=True)
```

Comment:
Actually I intended to use `sri_hash` specifically because it matched the spec for subresource integrity. That matches well with what I wanted (integrity of a subresource) and is the most easily available spec I know of that includes the algorithm of the hash in the string. I think that it's use is appropriate here.